README.md

```diff
@@ -2,85 +2,99 @@
 
 ## Pseudo-Random Element Generator
+
 [Pseudo random element generator module](./src/rand/) provides a set of traits and data structures that facilitate generating pseudo-random elements in the context of Miden VM and Miden rollup. The module currently includes:
 
-* `FeltRng`: a trait for generating random field elements and random 4 field elements.
-* `RpoRandomCoin`: a struct implementing `FeltRng` as well as the [`RandomCoin`](https://github.com/facebook/winterfell/blob/main/crypto/src/random/mod.rs) trait using RPO hash function.
-* `RpxRandomCoin`: a struct implementing `FeltRng` as well as the [`RandomCoin`](https://github.com/facebook/winterfell/blob/main/crypto/src/random/mod.rs) trait using RPX hash function.
-    
+- `FeltRng`: a trait for generating random field elements and random 4 field elements.
+- `RpoRandomCoin`: a struct implementing `FeltRng` as well as the [`RandomCoin`](https://github.com/facebook/winterfell/blob/main/crypto/src/random/mod.rs) trait using RPO hash function.
+- `RpxRandomCoin`: a struct implementing `FeltRng` as well as the [`RandomCoin`](https://github.com/facebook/winterfell/blob/main/crypto/src/random/mod.rs) trait using RPX hash function.
+
 ## Crate features
+
 This crate can be compiled with the following features:
 
-* `std` - enabled by default and relies on the Rust standard library.
-* `no_std` does not rely on the Rust standard library and enables compilation to WebAssembly.
+- `std` - enabled by default and relies on the Rust standard library.
+- `no_std` does not rely on the Rust standard library and enables compilation to WebAssembly.
 
 Both of these features imply the use of [alloc](https://doc.rust-lang.org/alloc/) to support heap-allocated collections.
 
-To compile with `no_std`, disable default features via `--no-default-features` flag.
+To compile with `no_std`, disable default features via `--no-default-features` flag or using the following command:
+
+```shell
+make build-no-std
+```
 
 ### AVX2 acceleration
+
 On platforms with [AVX2](https://en.wikipedia.org/wiki/Advanced_Vector_Extensions) support, RPO and RPX hash function can be accelerated by using the vector processing unit. To enable AVX2 acceleration, the code needs to be compiled with the `avx2` target feature enabled. For example:
+
 ```shell
-cargo make build-avx2
+make build-avx2
 ```
 
 ### SVE acceleration
-On platforms with [SVE](https://en.wikipedia.org/wiki/AArch64#Scalable_Vector_Extension_(SVE)) support, RPO and RPX hash function can be accelerated by using the vector processing unit. To enable SVE acceleration, the code needs to be compiled with the `sve` target feature enabled. For example:
+
+On platforms with [SVE](<https://en.wikipedia.org/wiki/AArch64#Scalable_Vector_Extension_(SVE)>) support, RPO and RPX hash function can be accelerated by using the vector processing unit. To enable SVE acceleration, the code needs to be compiled with the `sve` target feature enabled. For example:
```

Comment:
Question: why do we add angle brackets around the URL here?

Reply:
It  seems that my formatter adds it auto because of the `_(SVE)`, would you prefer to remove it?

Reply:
It doesn't seem like it is breaking anything. Let's keep as you have it.